assets/chezmoi.io/docs/reference/templates/azure-key-vault-functions/azureKeyVault.md

```diff
@@ -0,0 +1,15 @@
+# `azureKeyVault` *secret name* [*vault-name*]
+
+`azureKeyVault` returns a secret value retrieved from an
+[Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/).
+
+The mandatory `secret name` argument specifies the *name of the secret* to
+retrieve.
+
+The optional `vault name` argument specifies the *name of the vault*, if not set,
+the default vault name will be used.
+
+!!! warning
+
+    The current implementation will always return the latest version of the secret.
+    Defining a specific version of a secret is not supported.
```

Comment:
I would use `Obtaining` or `Retrieving` instead of `Defining`.

Reply:
Completed.